audb/core/info.py

```diff
@@ -28,8 +28,11 @@ def attachments(
     Returns:
```

Comment:
In both `numpy` and `pandas` they use the imported module when showing examples, e.g.:

```python
>>> d = {'col1': [1, 2], 'col2': [3, 4]}
>>> df = pd.DataFrame(data=d)
>>> df
   col1  col2
0     1     3
1     2     4
```

`pandas` does not show the `import pandas as pd` line, whereas `numpy` does start with `import numpy as np`.
As with `audb` we are not using another alias name when importing, I would argue we can skip the import to not clutter the examples.

The other advantage of writing `audb.info.attachments()` instead of `attachments()` is that a user can simply copy and paste the code and it works after a `import audb`, whereas when using single function names, they all would be needed to be imported explicitly, e.g. `from audb.info import attachments, author`, which I would not encourage.

Reply:
But I have to admit, that in our other packages, we have so far also most often written only the function name, e.g. https://audeering.github.io/audeer/api/audeer.common_directory.html. But I also remember, that I was always annoid that I cannot simply copy and paste the code for that reason.